test/test_helper.rb

```diff
@@ -69,5 +69,9 @@ def set_basic_auth(user, password)
       @request.env['CONTENT_TYPE'] = 'application/json'
       @request.env['HTTP_ACCEPT'] = 'application/json'
     end
+
+    def disable_webpack
```

Comment:
Functional tests will fail if the webpack assets were not compiled because the `webpack_asset_paths` function (in `views/layout/base.html.erb`) will attempt to read the `manifest.json` that does not exist. 
This stub allows running functional tests without downloading npm modules and compiling the assets - which is significant for reducing Jenkins' run time (`npm install` takes about 3-4 minutes per slave on Jenkins, so we only run it on the slaves doing integration testing)